packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -222,7 +216,7 @@ export const ribbonDividerStyle = (theme: Theme): string => {
   return mergeStyles({
     color: theme.palette.neutralQuaternaryAlt,
     margin: '0 -0.5rem',
```

Comment:
Noticed we removed some of the negative values for margins, which looks the right decision. Is the -0.5rem required here still?

Reply:
Yes, we still need it because this is a horizontal margin to make width of dividers between buttons smaller (and not have too much spase between the button and divider)